packages/core/src/scss/components/foundation/modal/_modal.scss

```diff
@@ -170,3 +167,37 @@
     display: block;
     visibility: visible;
 }
+
+// Specific styles for displaying cart summary on mobile
+body.has-activeModal {
+    overflow: auto;
```

Comment:
👍

Reply:
Thanks @valentindellangela. I just found out this bugfix broke BC Wordpress plug-in when opening a cart summary modal. So have to keep digging.